Header header

uint16 firmware
uint8 lococommand
uint16 locostatus
bool backFallDown
bool frontFallDown
time lastFallDownStamp
uint16 compass
uint16 gyroX
uint16 gyroY
uint16 gyroZ
uint8 linearX
uint8 linearY
uint8 angularVel
uint16 magnetoX
uint16 magnetoY
uint16 magnetoZ